x/wasm/ibctesting/chain.go

```diff
@@ -550,11 +550,11 @@ func (chain *TestChain) CreatePortCapability(scopedKeeper capabilitykeeper.Scope
 	_, ok := chain.App.ScopedIBCKeeper.GetCapability(chain.GetContext(), host.PortPath(portID))
 	if !ok {
 		// create capability using the IBC capability keeper
-		cap, err := chain.App.ScopedIBCKeeper.NewCapability(chain.GetContext(), host.PortPath(portID))
+		portCap, err := chain.App.ScopedIBCKeeper.NewCapability(chain.GetContext(), host.PortPath(portID))
```

Comment:
This and others: the `chain.go` originates from the ibc-go project. While the var names are more readable and verbose, we will have more conflicts when comparing the file to the original source for updates. Please consider also contributing this to ibc-go.